Source/WebGPU/WGSL/WGSL.h

```diff
@@ -209,6 +209,6 @@ struct PrepareResult {
 // These are not allowed to fail.
 // All failures must have already been caught in check().
 PrepareResult prepare(AST::ShaderModule&, const HashMap<String, PipelineLayout>&);
-PrepareResult prepare(AST::ShaderModule&, const String& entryPointName, const std::optional<PipelineLayout>&);
```

Comment:
How does the caller represent "the pipeline layout wasn't passed in from the site, instead, the pipeline layout should be generated from the WGSL source"? That's what this optional is supposed to represent.

Reply:
Hum, the reason I removed it was because it was never called with `nullopt`. I'll add it back.